### PR TITLE
Add a ModelName property to all devices

### PR DESCRIPTION
**Pull request by whot**
Some of the names are overlapping or at least hard to identify, it's useful to
be able to use the Wacom-assigned names (e.g. PTH-660) together with the
marketing name. Doubly so in the Intuos vs Bamboo-is-now-Intuos series.

Where there isn't an obvious model name it is NULL. This includes the various
ISDV4 devices that could use that as model name but in those cases that name
is encoded already in the human-readable name anyway.

### CODE REVIEW

File: data/intuos2-12x18.tablet [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [Device]
 Name=Wacom Intuos2 12x18
+ModelName=viceMatch=usb:056a:0045
```
[the remark this answers is not in the document]
Comment (whot) [reply data/intuos2-12x18.tablet]: 1212 for a 12x18 device? Is this correct?

[the remark this answers is not in the document]
Comment (jigpu) [reply data/intuos2-12x18.tablet]: *doh*, yeah, 1218

File: data/intuos-pro-2-l-wl.tablet [hunk 2]
```diff
@@ -44,6 +44,7 @@
 
 [Device]
 Name=Wacom Intuos Pro L WL
```
[the remark this answers is not in the document]
Comment (whot) [reply data/intuos-pro-2-l-wl.tablet]: Fixed in commit b930a2a, or a subsequent version of that if I have to force-push :)

**Comment by whot** [comment]
all done, thanks!

**Pull request merged**